presentation/src/main/java/com/yapp/growth/presentation/component/PlanzButtons.kt

```diff
@@ -59,23 +59,21 @@ fun PlanzButtonWithBack(
     onBackClick: () -> Unit,
 ) {
     Row(modifier = modifier.padding(horizontal = 16.dp)) {
```

Comment:
```suggestion
    Row(
        modifier = modifier.padding(horizontal = 16.dp),
        horizontalArrangement = Arrangement.spacedBy(10.dp)
    ) {
```

아래 Spacer 대신에 Row 의 horizontalArrangement 속성을 사용하는건 어때요?

Reply:
좋습니다! spacedBy 존재를 몰라서 이런 식으로 작성된 코드들 다시 살펴봐야겠어요!